bin/install-site.sh

```diff
@@ -143,12 +143,7 @@ EOF
         : # Do nothing on other Ubuntu versions, let's see if we need multiverse
     elif [ x"$DISTRIBUTION" = x"debian" ] && [ x"$DISTVERSION" = x"stretch" ]
     then
-        # Install the basic packages we require:
```

Comment:
Minor note but just want to tweak the commit message so that it auto-links correctly.

```
- See: #5a24d71
+ See: 5a24d71
```